tests/transformers/tests/generation/test_utils.py

```diff
@@ -45,41 +52,31 @@
         PreTrainedModel,
         SpeechEncoderDecoderModel,
     )
+    from transformers.cache_utils import DynamicCache
     from transformers.generation import (
         BeamSampleDecoderOnlyOutput,
         BeamSampleEncoderDecoderOutput,
         BeamSearchDecoderOnlyOutput,
         BeamSearchEncoderDecoderOutput,
-        BeamSearchScorer,
-        ConstrainedBeamSearchScorer,
         DisjunctiveConstraint,
-        ForcedBOSTokenLogitsProcessor,
-        ForcedEOSTokenLogitsProcessor,
+        GenerateBeamDecoderOnlyOutput,
+        GenerateBeamEncoderDecoderOutput,
+        GenerateDecoderOnlyOutput,
         GenerateEncoderDecoderOutput,
         GreedySearchDecoderOnlyOutput,
         GreedySearchEncoderDecoderOutput,
-        HammingDiversityLogitsProcessor,
-        InfNanRemoveLogitsProcessor,
         LogitsProcessorList,
         MaxLengthCriteria,
         MinLengthLogitsProcessor,
-        NoBadWordsLogitsProcessor,
-        NoRepeatNGramLogitsProcessor,
         PhrasalConstraint,
-        RepetitionPenaltyLogitsProcessor,
         SampleDecoderOnlyOutput,
         SampleEncoderDecoderOutput,
         StoppingCriteria,
         StoppingCriteriaList,
-        TemperatureLogitsWarper,
-        TopKLogitsWarper,
-        TopPLogitsWarper,
     )
     from transformers.generation.candidate_generator import AssistedCandidateGenerator, CandidateGenerator
-    from transformers.generation.streamers import BaseStreamer
-
-torch_device = "hpu"
-adapt_transformers_to_gaudi()
+    from transformers.generation.utils import _speculative_sampling
+    from transformers.streamers import BaseStreamer
```

Comment:
```suggestion
    from transformers.generation.streamers import BaseStreamer
```
@malkomes fix the import!

Reply:
if you re-base to main this is already there

Reply:
I'm on it